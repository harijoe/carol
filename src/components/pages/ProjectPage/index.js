import React from 'react'
import PropTypes from 'prop-types'

import { MainLayout, MainWrapper, InnerWrapper } from 'components'
import { ProjectDetails } from 'containers'

const ProjectPage = props =>
  <MainLayout>
    <MainWrapper>
      <InnerWrapper>
        <ProjectDetails id={props.params.projectId} />
      </InnerWrapper>
    </MainWrapper>
  </MainLayout>

ProjectPage.propTypes = {
  params: PropTypes.shape({
    projectId: PropTypes.string,
  }),
}

export default ProjectPage
