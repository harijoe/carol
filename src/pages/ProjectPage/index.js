import React from 'react'
import PropTypes from 'prop-types'

import MainLayout from 'layouts/MainLayout'
import MainWrapper from 'components/MainWrapper'
import ProjectDetails from 'containers/ProjectDetails'

const ProjectPage = props =>
  <MainLayout>
    <MainWrapper paddingTop="m">
      <ProjectDetails id={props.match.params.projectId} />
    </MainWrapper>
  </MainLayout>

ProjectPage.propTypes = {
  match: PropTypes.shape({
    params: PropTypes.shape({
      projectId: PropTypes.string,
    }),
  }),
}

export default ProjectPage
