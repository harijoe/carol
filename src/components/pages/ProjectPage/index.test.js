import React from 'react'
import { shallow } from 'enzyme'
import ProjectPage from './'

it('renders ProjectPage', () => {
  expect(shallow(<ProjectPage params={{ projectId: '1' }} />)).toMatchSnapshot()
})
