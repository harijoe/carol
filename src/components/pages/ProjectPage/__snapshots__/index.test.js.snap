// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ProjectPage 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <ProjectPage
    params={
      Object {
        "projectId": "1",
      }
    }
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": null,
    "nodeType": "function",
    "props": Object {
      "children": <Mock
        paddingTop="m"
      >
        <Mock
          id="1"
        />
      </Mock>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": null,
      "nodeType": "function",
      "props": Object {
        "children": <Mock
          id="1"
        />,
        "paddingTop": "m",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": null,
        "nodeType": "function",
        "props": Object {
          "id": "1",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      "type": [Function],
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": null,
      "nodeType": "function",
      "props": Object {
        "children": <Mock
          paddingTop="m"
        >
          <Mock
            id="1"
          />
        </Mock>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": null,
        "nodeType": "function",
        "props": Object {
          "children": <Mock
            id="1"
          />,
          "paddingTop": "m",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": null,
          "nodeType": "function",
          "props": Object {
            "id": "1",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        "type": [Function],
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
    "disableLifecycleMethods": true,
  },
}
`;
